packages/pilot/src/containers/RecipientTable/index.js

```diff
@@ -139,6 +140,7 @@ const RecipientTable = ({
                 <Pagination
                   currentPage={pagination.offset}
```

Comment:
Acho melhor fazer um destructuring na linha 39 desse arquivo e utilizar `offset` diretamente.
```
  pagination: {
    total,
    offset
  },
```
```suggestion
                  currentPage={offset}
```

Reply:
Feito :+1: